lib/ddf-definitions/concept.js

```diff
@@ -54,6 +56,30 @@ class Concept {
     return this.getAllData()
       .map(record => record.concept);
   }
+
+  getTranslationsData(onTranslationsReady) {
+    const transActions = _.compact(_.flattenDeep(
+      this.fileDescriptors
+        .map(fileDescriptor =>
+          fileDescriptor.getExistingTranslationDescriptors()
+            .map(translationDesc => onTranslationLoaded => {
+              const transCollectionName = `translation-${translationDesc.file}`;
+
+              this.db
```

Comment:
this.db.fillCollection(transCollectionName, translationDesc.fullPath, err => {
        translationDesc.content = this.db.getCollection(transCollectionName);
        onTranslationLoaded(err);
 }, true);	
})